test/release/examples/benchmarks/shootout/chameneosredux.chpl

```diff
@@ -74,6 +74,12 @@ proc simulate(numChameneos) {
   // print information about the population once we're done
   //
   printInfo(chameneos);
+
+  //
+  // hasta las chameneos, baby.
+  //
+  destroyChameneos(chameneos);
+  delete meetingPlace;
```

Comment:
Tom, does a similar change need to be done to the tests in test/studies/shootout/chameneos/bradc and /lydia?  That's where this test comes from in the first place